.eleventy.js

```diff
@@ -31,6 +32,11 @@ module.exports = function (eleventyConfig) {
     return content;
   });
 
+  // add `date` filter
+  eleventyConfig.addFilter('date', function (date, dateFormat) {
```

Comment:
Maybe I'm being blind but I can't seem to find where this filter is used.

Reply:
Right good catch! My bad. I forgot to remove it. This filter isn't necessary!